core-services/prow/02_config/_config.yaml

```diff
@@ -910,6 +910,7 @@ tide:
     - openshift-kni/performance-addon-operators
     - openshift-kni/cnf-features-deploy
     - openshift-priv/ci-experiment-origin
+    - openshift/extended-platform-tests
```

Comment:
We should also add this repo to the other long list, the one that configures tide for the non-master branches.